web/packages/teleport/src/Locks/types.ts

```diff
@@ -54,6 +54,10 @@ export type AllowedTargets =
   | 'device';
 
 export type TableData = {
+  // targetValue is not displayed in the table, but is is the value
+  // that will be used when creating the lock target
+  targetValue: string;
+  // these values are shown in the UI (each key is a separate column)
   [key: string]: string;
```

Comment:
If the types in `targetDataFilters` were more strict, this would cause an error - `labels` is not `string`.